secure-biometric-interface-integrator/lib/secureBiometricInterface.js

```diff
@@ -25,7 +26,9 @@ import {
   DeviceState,
   DeviceStateStatus,
   DEFAULT_PROPS,
+  ErrorCode,
 } from "./standardConstant";
+import { isAxiosError } from "axios";
```

Comment:
IsAxiousError is not used, can be removed

Reply:
fix in this commit [5ebb03c](https://github.com/mosip/esignet-plugins/pull/31/commits/5ebb03c7250691483cd36883d9cac477a1431126)